reportTemplatesSrc/studentReport.js

```diff
@@ -41,6 +42,7 @@ class StudentReport extends Component {
       students,
       activitiesHeaders,
       schoolYear,
+      student, 
```

Comment:
Looks like this is unused, and if I remember the structure of the object, I think its an undefined field.